src/main/java/equipments/Equipment.java

```diff
@@ -0,0 +1,6 @@
+package equipments;
+
+public interface Equipment {
+    public String equipmentType();
```

Comment:
methods should contain actions in their name- change this to `getEquipmentType`

Reply:
done.